core/string/ustring.h

```diff
@@ -299,6 +301,8 @@ class [[nodiscard]] String {
 	_FORCE_INLINE_ const char32_t *ptr() const { return _cowdata.ptr(); }
 	_FORCE_INLINE_ const char32_t *get_data() const { return ptr() ? ptr() : &_null; }
 
+	// Note: This returns the size of the buffer, which includes the terminating NUL character.
```

Comment:
It's easy to assume `size of the buffer` is size in bytes, while it is the number of `char32_t`s. So it might be worth specifying it.

Reply:
Good point. I adjusted the wording, I hope now it's a bit clearer.